src/server/common/model/answer/kept-separately/kept-separately.js

```diff
@@ -0,0 +1,31 @@
+import { RadioButtonAnswer } from '../radio-button/radio-button.js'
+/** @import {RadioButtonConfig} from '../radio-button/radio-button.js' */
+
+/** @type {RadioButtonConfig} */
+const keptSeparatelyConfig = {
+  payloadKey: 'keptSeparately',
+  options: {
+    yes: { label: 'Yes' },
+    no: { label: 'No' }
+  },
+  errors: {
+    emptyOptionText: 'Select if the incoming cattle will be kept separately'
+  },
+  layout: 'inline'
+}
+
+/**
+ * export @typedef {'yes'|'no'} KeptSeparatelyData
+ * @typedef {{ keptSeparately: KeptSeparatelyData }} KeptSeparatelyPayload
+ */
+
+/** @augments {RadioButtonAnswer<KeptSeparatelyPayload>} */
+export class KeptSeparatelyAnswer extends RadioButtonAnswer {
+  get config() {
```

Comment:
Once we've updated the config strategy, we can replace this with:

```
export class KeptSeparatelyAnswer extends RadioButtonAnswer {
  static config = keptSeparatelyConfig
}
```

(or just inline the config, as we have done elsewhere)